bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/core/TelldusCoreBridgeHandler.java

```diff
@@ -82,10 +85,13 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     @Override
     public void dispose() {
         logger.debug("Telldus Core Handler disposed.");
+        TelldusDeviceController deviceController = this.deviceController;
         if (deviceController != null) {
             deviceController.dispose();
             deviceController = null;
```

Comment:
As a consequence of introducing the local variable, you need to explicitly set the instance variable to null here:
```suggestion
            this.deviceController = null;
```